apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant/Defendant.tsx

```diff
@@ -1,504 +1,91 @@
+import { Box } from '@island.is/island-ui/core'
 import * as constants from '@island.is/judicial-system/consts'
 import { getStandardUserDashboardRoute } from '@island.is/judicial-system/consts'
-import {
-  CrimeScene,
-  CrimeSceneMap,
-  IndictmentSubtypeMap,
-} from '@island.is/judicial-system/types'
-import { core, errors, titles } from '@island.is/judicial-system-web/messages'
+import { core, titles } from '@island.is/judicial-system-web/messages'
 import {
   FormContentContainer,
   FormContext,
   FormFooter,
   PageHeader,
   PageLayout,
   PageTitle,
-  SectionHeading,
   UserContext,
 } from '@island.is/judicial-system-web/src/components'
-import {
-  Case,
-  CaseOrigin,
-  Defendant as TDefendant,
-  IndictmentSubtype,
-  UpdateDefendantInput,
-} from '@island.is/judicial-system-web/src/graphql/schema'
 import {
   useCase,
   useDefendants,
-  useIndictmentCounts,
 } from '@island.is/judicial-system-web/src/utils/hooks'
-import { getDefaultDefendantGender } from '@island.is/judicial-system-web/src/utils/utils'
 import { isDefendantStepValidIndictments } from '@island.is/judicial-system-web/src/utils/validate'
 
-import { DefendantInfo, ProsecutorSection } from '../../components'
-import { getIndictmentIntroductionAutofill } from '../Indictment/Indictment'
-import { getIncidentDescription } from '../Indictment/lib/getIncidentDescription'
-import { LokeNumberList } from './LokeNumberList/LokeNumberList'
-import { PoliceCaseInfo } from './PoliceCaseInfo/PoliceCaseInfo'
-import { usePoliceCaseInfoQuery } from './policeCaseInfo.generated'
-import { defendant } from './Defendant.strings'
-
-export interface PoliceCase {
-  number: string
-  subtypes?: IndictmentSubtype[]
-  place?: string
-  date?: Date
-}
-
-const compareCrimeSceneDates = <T extends string | number>(
-  a: T | undefined | null,
-  b: T | undefined | null,
-) => {
-  // We want missing dates to be at the end of the list
-  if (a === undefined || a === null) {
-    return b === undefined || b === null ? 0 : 1
-  }
-
-  if (b === undefined || b === null) {
-    return -1
-  }
-
-  return a !== b ? (a < b ? -1 : 1) : 0
-}
-
-const getPoliceCases = (theCase: Case): PoliceCase[] =>
-  theCase.policeCaseNumbers && theCase.policeCaseNumbers.length > 0
-    ? theCase.policeCaseNumbers.map((policeCaseNumber) => ({
-        number: policeCaseNumber,
-        subtypes:
-          theCase.indictmentSubtypes &&
-          theCase.indictmentSubtypes[policeCaseNumber],
-        place:
-          theCase.crimeScenes && theCase.crimeScenes[policeCaseNumber]?.place,
-        date:
-          theCase.crimeScenes &&
-          theCase.crimeScenes[policeCaseNumber]?.date &&
-          new Date(theCase.crimeScenes[policeCaseNumber].date),
-      }))
-    : [{ number: '' }]
+import { ProsecutorSection } from '../../components'
+import { DefendantList } from './DefendantList/DefendantList'
+import { PoliceCaseList } from './PoliceCaseList/PoliceCaseList'
```

Comment:
The list of defendants and police cases have been extracted into separate components to simplify the defendant component.